crates/viewer/re_blueprint_tree/tests/range_selection_test.rs

```diff
@@ -0,0 +1,126 @@
+    let mut test_context = TestContext::default();
+
+    test_context.register_view_class::<re_view_spatial::SpatialView3D>();
+
+    for i in 0..=10 {
+        test_context.log_entity(format!("/entity{i}").into(), add_point_to_chunk_builder);
+    }
+
+    test_context.setup_viewport_blueprint(|_, blueprint| {
+        blueprint.add_views(
+            std::iter::once(ViewBlueprint::new(
+                re_view_spatial::SpatialView3D::identifier(),
+                RecommendedView::root(),
+            )),
+            None,
+            None,
+        );
+    });
+
+    let mut blueprint_tree = BlueprintTree::default();
+
+    // set the current timeline to the timeline where data was logged to
+    test_context.set_active_timeline(Timeline::new_sequence("frame_nr"));
+
+    let mut harness = test_context
+        .setup_kittest_for_rendering()
+        .with_size(Vec2::new(400.0, 500.0))
+        .build(|ctx| {
+            // We must create a side panel here (instead of the default central panel, as
+            // `list_item::LabelContent`'s sizing behave differently there.
+            egui::SidePanel::left("blueprint_tree")
+                .default_width(400.0)
+                .show(ctx, |ui| {
+                    test_context.run(&ui.ctx().clone(), |viewer_ctx| {
+                        let blueprint = ViewportBlueprint::try_from_db(
+                            viewer_ctx.store_context.blueprint,
+                            viewer_ctx.blueprint_query,
+                        );
+
+                        // expand the view
+                        //TODO(ab): with Rust 1.83, use `break_value().expect()` instead.
+                        let ControlFlow::Break(view_id) =
+                            blueprint.visit_contents(&mut |contents, _| match contents {
+                                Contents::View(id) => VisitorControlFlow::Break(*id),
+                                Contents::Container(_) => VisitorControlFlow::Continue,
+                            })
+                        else {
+                            panic!("A view we know exists was not found");
+                        };
+
+                        re_context_menu::collapse_expand::collapse_expand_view(
+                            viewer_ctx,
+                            &view_id,
+                            blueprint_tree.collapse_scope(),
+                            true,
+                        );
+
+                        blueprint_tree.show(viewer_ctx, &blueprint, ui);
+                    });
+
+                    test_context.handle_system_commands();
+                });
+        });
+
+    harness.run();
+
+    let node0 = harness.get_by_label("entity0");
+    node0.click();
+
+    harness.run();
+    let node2 = harness.get_by_label("entity2");
+    node2.key_down(Key::Shift);
+    node2.click();
+    node2.key_up(Key::Shift);
+
+    //TODO(ab): remove this when https://github.com/emilk/egui/pull/5693 is landed/released
```

Comment:
I think I didn't catch that on previous reviews: how we usually do this is by writing this as

```suggestion
    //TODO(emilk/egui#5693): remove this when https://github.com/emilk/egui/pull/5693 is landed/released
```
this way our script for orphaned todo notes can pick it up